src/asciidoc/docs/develop/editor.adoc

```diff
@@ -1,123 +1,156 @@
-= The Koji editor
+= The Koji Code Editor
 :page-slug: editor
-:page-description: Guide for the Koji editor, a browser-based IDE for developing and testing Koji templates.
+:page-description: Guide for the Koji Code Editor, a browser-based IDE for developing and testing a first-generation Koji.
 :includespath: ../_includes
 
+The Koji Code Editor is a browser-based integrated development environment (IDE) that you can use to develop and test a first-generation Koji.
+The Code Editor provides easy access to the following:
+[.singlespaced]
+* All of your project's code
+* Visual Customization Controls (VCCs) generated from the project's code
+* Production tools and assets, such as the Koji database
+* Tools to publish and deploy your Koji
 
-The Koji editor is a browser-based integrated development environment (IDE).
-You can use the Koji editor to create application and game templates.
-The editor contains all of your project's code, Visual Customization Controls (VCCs) generated from the project's code, interfaces for publishing and deployment management, and interfaces to other production tools such as the Koji database.
 
 == Overview
 
-The Koji editor is divided into these sections:
+To create a new Koji, you must set up a project and open it in the Koji Code Editor.
+[.singlespaced]
+. Create a project.
++
+include::{includespath}/steps-codeeditor.adoc[tag=createproject-details-collapsible]
```

Comment:
Only the steps are not are directly related to the topic are collapsible. In this case, the step of creating a project does not involve the code editor.

Also the "Create a project" step is an include that we'll very likely use in a lot of places. It can be irritating to sift through the same expanded step over and over as readers read multiple topics that include that step.

Reply:
If a Koji project is forkable, doesn't it appear in the For Developers list too? If not, where can they find a list of the forkable projects?